### PR TITLE
doc: Doc fixes post 2.2

### PR DESCRIPTION
**Pull request by umapraseeda**
Doc fixes post 2.2

### CODE REVIEW

File: doc/nrf/external_comp/bt_fast_pair.rst [hunk 1]
```diff
@@ -88,7 +88,7 @@ The Fast Pair service implementation provides API to generate the advertising da
   Make sure that these functions are called by the application from the cooperative context to ensure that not discoverable advertising data generation is not preempted by an Account Key write operation from a connected Fast Pair Seeker.
   Account Keys are used to generate not discoverable advertising data.
   Special battery data can be included in not discoverable advertising packet using the Fast Pair Battery Notification extension.
-  To use this extension, you have to:
+  To use this extension, you have to esnure the following:
```
Comment (peknis) [inline doc/nrf/external_comp/bt_fast_pair.rst:91]:
```suggestion
  To use this extension, ensure the following:
```

File: include/modem/modem_slm.h [hunk 2]
```diff
@@ -46,7 +46,7 @@ enum at_cmd_state {
  * @typedef slm_data_handler_t
  *
  * Handler to handle data received from SLM, which could be AT response, AT notification
- * or or simply raw data (e.g. DFU image).
+ * or simply raw data (e.g. DFU image).
```
Comment (peknis) [inline include/modem/modem_slm.h:49]:
```suggestion
 * or simply raw data (for example DFU image).
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by alexsven — APPROVED** [review]

**Review by MarkusLassila — APPROVED** [review]
SLM is changes are OK.

**Review by simensrostad — APPROVED** [review]

**Review by divipillai — APPROVED** [review]

File: applications/asset_tracker_v2/doc/sensor_module.rst [hunk 3]
```diff
@@ -7,7 +7,7 @@ Sensor module
    :local:
    :depth: 2
 
-The sensor module interacts with external sensors present on the `Thingy:91 <Thingy:91 product page>`_.
+The sensor module interacts with external sensors present on the `Thingy:91 <Thingy\:91 product page_>`_.
```
Comment (MarekPieta) [inline applications/asset_tracker_v2/doc/sensor_module.rst:10]: Was the `\` added on purpose?

**Review by MarekPieta — APPROVED** [review]

File: applications/asset_tracker_v2/doc/sensor_module.rst (hunk 3 above)
Comment (peknis) [reply applications/asset_tracker_v2/doc/sensor_module.rst:10]: That must be just a mishap. No backslash needed there.

**Review by VeijoPesonen — APPROVED** [review]

**Review by SeppoTakalo — APPROVED** [review]

**Review by maje-emb — APPROVED** [review]
Radio Test, nrf_dm and dm look OK.

File: applications/asset_tracker_v2/doc/sensor_module.rst (hunk 3 above)
Comment (divipillai) [reply applications/asset_tracker_v2/doc/sensor_module.rst:10]: The link don't work with the combination of these special characters, and adding \ would fix the issue.
May be to override the special meaning provided to these markup characters.
https://docutils.sourceforge.io/docs/user/rst/quickref.html#escaping

Comment (peknis) [reply applications/asset_tracker_v2/doc/sensor_module.rst:10]: Oh, good point Divya. I missed that.

**Review by kkasperczyk-no — APPROVED** [review]

**Review by adsz-nordic — APPROVED** [review]

File: samples/openthread/coap_server/README.rst [hunk 4]
```diff
@@ -71,8 +71,6 @@ Specify the corresponding file names in the :makevar:`OVERLAY_CONFIG` option whe
 See :ref:`cmake_options` for instructions on how to add this option.
 For more information about using configuration overlay files, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
 
-The following configuration files are available:
```
Comment (canisLupus1313) [inline samples/openthread/coap_server/README.rst:74]: Why have you removed this line?

Comment (umapraseeda) [reply samples/openthread/coap_server/README.rst:74]: Even though the introductory line says "The following configuration files are available:" there is only one file mentioned below. Are there any more files to be documented? If there is only file, we might not even need a bulleted list and may be we can rephrase this as ":file:`overlay-logging.conf` is a configuration file that enables logging using RTT.  For additional options, refer to :ref:`RTT logging <ug_logging_backends_rtt>`."

**Review by jayteemo — APPROVED** [review]

File: samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst [hunk 5]
```diff
@@ -29,7 +29,7 @@ Features
 
 This sample implements or demonstrates the following features:
 
-* Error-tolerant use of the `nRF Cloud MQTT API`_ using the :ref:`nrf_modem <nrfxlib:nrf_modem>` and :ref:`nrf_cloud <lib_nrf_cloud>` libraries.
+* Error-tolerant use of the `nRF Cloud MQTT API`_ using the :ref:`nrf_modem` and :ref:`lib_nrf_cloud` library.
```
Comment (plskeggs) [inline samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst:32]:
```suggestion
* Error-tolerant use of the `nRF Cloud MQTT API`_ using the :ref:`nrf_modem` and :ref:`lib_nrf_cloud` libraries.
```
This sentence is referring to two libraries, so the word should be plural.

**Review by plskeggs — CHANGES_REQUESTED** [review]

File: samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst (hunk 5 above)
Comment (umapraseeda) [reply samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst:32]: @plskeggs please see the output here: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-10344/nrf/samples/nrf9160/nrf_cloud_mqtt_multi_service/README.html#features. :ref:\`nrf_modem\` already expands to Modem library.

Comment (plskeggs) [reply samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst:32]: I see.  Thanks.

File: doc/nrf/protocols/matter/end_product/certification.rst [hunk 6]
```diff
@@ -121,7 +121,7 @@ Certification document templates from Nordic Semiconductor
 
   .. note::
      Nordic Semiconductor provides just an example answers based on the reference SDK applications, which may guide you on how to answer to certain platform-related questions.
-     You are still solely responsibility for filling original documents according to a product specifiction.
+     You are still solely responsibility for filling original documents according to a product specification.
```
Comment (glarsennordic) [inline doc/nrf/protocols/matter/end_product/certification.rst:124]:
```suggestion
     You are still solely responsible for filling original documents according to a product specification.
```

**Review by glarsennordic — APPROVED** [review]
Additional minor typo spotted, but I don't want to hold up merge

Otherwise, LGTM

**Review by trantanen — APPROVED** [review]
Location library and modem shell parts OK

**Review by sachinthegreen — APPROVED** [review]

File: doc/nrf/protocols/matter/end_product/certification.rst (hunk 6 above)
Comment (umapraseeda) [reply doc/nrf/protocols/matter/end_product/certification.rst]: fixed this. Thanks!

File: doc/nrf/external_comp/bt_fast_pair.rst (hunk 1 above)
Comment (umapraseeda) [reply doc/nrf/external_comp/bt_fast_pair.rst]: fixed

File: include/modem/modem_slm.h (hunk 2 above)
Comment (umapraseeda) [reply include/modem/modem_slm.h]: fixed

**Review by peknis — APPROVED** [review]
